### PR TITLE
Updated to @prismicio/client and docs to make theme command launch with content

### PR DESCRIPTION
**Pull request by pvpg**
- Added documents for the theme command
- Changed the README and package.json description/info
- Updated prismic-javascript to @prismicio/client
- Updated toolbar script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,39 @@
-# Prismic Next.js Blog Site Example
-> [Next.js](https://nextjs.org/) example blog project with content managed in [Prismic](https://prismic.io)
+# Prismic & NextJS Example Blog
```
Comment (samlfair) [inline README.md]: It's "Next.js", not "NextJS".

File: README.md [hunk 2]
```diff
@@ -1,42 +1,19 @@
...
-
-[Prismic + Next.js documentation](https://prismic.io/docs/reactjs/getting-started/prismic-nextjs).
-
-### License
+## License
 
 This software is licensed under the Apache 2 license, quoted below.
 
-Copyright 2019 Prismic (https://prismic.io).
+Copyright 2021 [Prismic](http://prismic.io).
```
Comment (levimykel) [inline README.md]: Use `https` for the Prismic site

File: components/PrismicScript.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { apiEndpoint } from 'prismic-configuration'
 const PrismicScript = () => {
   const [, repoName] = apiEndpoint.match(/https?:\/\/([^.]+)?\.(cdn\.)?.+/);
   return (
-    <script async defer src={`"https://static.cdn.prismic.io/prismic.min.js?repo=${repoName}&new=true"`} />
+    <script async defer src={`"https://static.cdn.prismic.io/prismic.js?new=true&repo=${repoName}"`}></script>
```
Comment (levimykel) [inline components/PrismicScript.js]: Use a self closing tag if there are no children in JSX: ` />`

**Review by levimykel — COMMENTED** [review]
@PaulinaVPG I left a couple of comments here. And it appears that there is a conflict that needs to be fixed before this can be merged.

**Pull request merged**